core-services/prow/02_config/quay/quay-bridge-operator/_pluginconfig.yaml

```diff
@@ -0,0 +1,50 @@
+approve:
```

Comment:
Maybe you also want to enable TIde (automated merges) in a `_prowconfig.yaml` file? Probably easiest to copy another repos config.

Reply:
Interesting, I copied it from https://github.com/openshift/release/blob/master/core-services/prow/02_config/openshift/_pluginconfig.yaml, thanks for heads up!